### PR TITLE
[make:reset-password] increase password strength & check for comprimised password

### PR DESCRIPTION
**Pull request by Spomky**
For the password reset process:
* [x] Adds the `NotCompromisedPassword` and `PasswordStrength` constraints to the password reset form
* [x] Set a minimal password length to 12 instead of 6

### CODE REVIEW

File: src/Resources/skeleton/resetPassword/ChangePasswordFormType.tpl.php [hunk 1]
```diff
@@ -22,11 +22,13 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                             'message' => 'Please enter a password',
                         ]),
                         new Length([
-                            'min' => 6,
+                            'min' => 12,
                             'minMessage' => 'Your password should be at least {{ limit }} characters',
                             // max length allowed by Symfony for security reasons
                             'max' => 4096,
                         ]),
+                        new PasswordStrength(),
```
Comment (94noni) [inline src/Resources/skeleton/resetPassword/ChangePasswordFormType.tpl.php:30]: I like this 👍🏻

**Comment by 94noni** [comment]
perhaps split into 2 PRs ?
one related to login password and other to throttling

**Comment by Spomky** [comment]
> perhaps split into 2 PRs ? one related to login password and other to throttling

Yes indeed. I will change it.
The throttling part is not complete as I should add the symfony/rate-limiter as a dependency.

**Review by jrushlow — APPROVED** [review]
Awesome! thank you @Spomky for this PR

**Pull request merged**